src/Commands/IndexProductsCommand.php

```diff
@@ -28,17 +26,10 @@ public function handle()
         IndexBeforeEvent::dispatch($this);
 
         $productModel = config('rapidez.models.product');
-        $storeModel = config('rapidez.models.store');
-        $stores = $this->argument('store') ? $storeModel::where('store_id', $this->argument('store'))->get() : $storeModel::all();
```

Comment:
So we can not run the indexer per store anymore?

Reply:
oops, fixed along with the setStore/getStore functionality improvement 😇